dirty-flag/README.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: pattern
+title: Dirty Flag
+folder: dirty-flag
+permalink: /patterns/dirty-flag/
+categories: Other
+tags:
+ - Java
+ - Difficulty-Easy
+ - Performance
+---
+
+## Intent
+To avoid expensive re-acquisition of resources. The resources retain their identity, are kept in some
+fast-access storage, and are re-used to avoid having to acquire them again.
+
+![alt text](./etc/dirty-flag.png "Dirty Flag")
```

Comment:
The class diagram should be made with ObjectAid eclipse plugin. More about it in wiki

Reply:
Done.